metis-normalization/src/main/java/eu/europeana/normalization/dates/cleaning/Cleaner.java

```diff
@@ -12,54 +12,79 @@
  * <p>
  * This class also detects some patterns frequently used for adding notes to the dates, and removes such notes.
  * </p>
+ * <p>Before any pattern is ran, multiple space characters are replaced by a single literal space character and the string is
+ * trimmed.</p>
  */
 public class Cleaner {
```

Comment:
I don't see the suggested names as to be more clarifying to be honest.
Other names I could think of perhaps are `DateFieldCleaner` , `DateFieldSanitizer`, `DatePropertyCleaner` , `DatePropertySanitizer`.
What do you think?

Reply:
The reason why I came up with this word **amendment** or **amender** or **amend** is about the definiiton _"An alteration or change for the better; correction of a fault or of faults; reformation of life by quitting vices." after understanding the purpose of the class this could suit better the name."_ actually the code you wrote is correcting and improving for better is not only cleaning.

Reply:
after our offline discussion, I think that `DateFieldSanitizer` would suit better.